<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nhập thông tin cho chứng chỉ:</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>500</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #000000;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 18px;
    font-size: 16px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #2c3e50;
}

QPushButton:pressed {
    background-color: #1a252f;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>30</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Tên chứng chỉ (Cer name):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>-387</x>
     <y>-14</y>
     <width>1291</width>
     <height>661</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../image/resource.qrc">:/new/image/background.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="cerName">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ecf0f1;
    border: 2px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 18px;
    color: #2c3e50;
}

QLineEdit:focus {
    border: 2px solid #3498db;
    background-color: #ffffff;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>80</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Tên định danh (CN):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CNName">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ecf0f1;
    border: 2px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 18px;
    color: #2c3e50;
}

QLineEdit:focus {
    border: 2px solid #3498db;
    background-color: #ffffff;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>130</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Tên tổ chức (Org):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OrgName">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ecf0f1;
    border: 2px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 18px;
    color: #2c3e50;
}

QLineEdit:focus {
    border: 2px solid #3498db;
    background-color: #ffffff;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>180</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Mã quốc gia (Country):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CountryName">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ecf0f1;
    border: 2px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 18px;
    color: #2c3e50;
}

QLineEdit:focus {
    border: 2px solid #3498db;
    background-color: #ffffff;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Ngày hiệu lực:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valiDay">
   <property name="geometry">
    <rect>
     <x>371</x>
     <y>230</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ecf0f1;
    border: 2px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 18px;
    color: #2c3e50;
}

QLineEdit:focus {
    border: 2px solid #3498db;
    background-color: #ffffff;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>280</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Loại chứng chỉ:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CerLoai">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CA Certificate</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>End-Entity Certificate</string>
    </property>
   </item>
  </widget>
  <zorder>label_18</zorder>
  <zorder>buttonBox</zorder>
  <zorder>label_16</zorder>
  <zorder>cerName</zorder>
  <zorder>label_17</zorder>
  <zorder>CNName</zorder>
  <zorder>label_19</zorder>
  <zorder>OrgName</zorder>
  <zorder>label_20</zorder>
  <zorder>CountryName</zorder>
  <zorder>label_21</zorder>
  <zorder>valiDay</zorder>
  <zorder>label_22</zorder>
  <zorder>CerLoai</zorder>
 </widget>
 <resources>
  <include location="../image/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
